app/components/floating_text_input_label/index.tsx

```diff
@@ -180,20 +106,10 @@ const FloatingTextInput = forwardRef<FloatingTextInputRef, FloatingTextInputProp
     const shouldShowError = (!focused && error);
     const onPressAction = !isKeyboardInput && editable && onPress ? onPress : undefined;
 
-    const combinedContainerStyle = useMemo(() => {
-        const res = [styles.container];
-        if (multiline) {
-            res.push({height: 100 + (2 * BORDER_DEFAULT_WIDTH)});
-        }
-        res.push(containerStyle);
-        return res;
-    }, [styles, containerStyle, multiline]);
-
-    const combinedTextInputStyle = useMemo(() => {
-        const res: StyleProp<TextStyle> = [styles.textInput];
+    const combinedTextInputContainerStyle = useMemo(() => {
+        const res: StyleProp<TextStyle> = [styles.textInputContainer, textInputContainerStyle, {height: textInputContainerHeight}];
```

Comment:
There is a change of behavior here, since now the `height` cannot be overwritten by `textInputContainerStyle`. Is this intended? If not, let's keep the correct order.

Also, I see that we are pushing into this style the border width and color. I wonder if those should be overwritten by the `textInputContainerStyle` or not.

Reply:
The height is being set to textInputContainerHeight, which is a variable that contains the computed value from the base styles and the extra styles provided, which allows the input overall styles to adapt properly to different heights

```ts
const textInputContainerHeight = useMemo(() => getInputContainerHeight(multiline, textInputContainerStyle), [multiline, textInputContainerStyle]);
```
```ts
export const getInputContainerHeight = (
    multiline: Boolean, textInputContainerOverrideStyle: ViewStyle | undefined,
) => {
    let height = multiline ? MULTILINE_INPUT_CONTAINER_HEIGHT : DEFAULT_INPUT_CONTAINER_HEIGHT;

    if (textInputContainerOverrideStyle?.height) {
        height = textInputContainerOverrideStyle.height as number || 0;
    }

    return height;
};
```